namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-demo/resources/irsa.tf

```
@@ -0,0 +1,91 @@
+locals {
+  sqs_queues_ap = {
+    "Digital-Prison-Services-dev-hmpps_audit_queue" = "hmpps-audit-demo",
+  }
+  sqs_queues_cas2 = {
+    "Digital-Prison-Services-dev-hmpps_audit_queue" = "hmpps-audit-demo",
+  }
+  sqs_queues_ta = {
+    "Digital-Prison-Services-dev-hmpps_audit_queue" = "hmpps-audit-demo",
+  }
+  sns_topics = {
+    "cloud-platform-Digital-Prison-Services-e29fb030a51b3576dd645aa5e460e573" = "hmpps-domain-events-demo"
+  }
```

see this line as example on our test environment - we only have event and audit queues for dev, preprod and prod as far as i can see for all services.
 - i think these need to point to dev audit and domain event queues ot get this functioning.
```suggestion
  sqs_queues_ap = {
    "Digital-Prison-Services-dev-hmpps_audit_queue" = "hmpps-audit-dev",
  }
  sqs_queues_cas2 = {
    "Digital-Prison-Services-dev-hmpps_audit_queue" = "hmpps-audit-dev",
  }
  sqs_queues_ta = {
    "Digital-Prison-Services-dev-hmpps_audit_queue" = "hmpps-audit-dev",
  }
  sns_topics = {
    "cloud-platform-Digital-Prison-Services-e29fb030a51b3576dd645aa5e460e573" = "hmpps-domain-events-dev"
  }
```